frontend/src/components/Calendar/calendar.tsx

```diff
@@ -0,0 +1,63 @@
+import { faArrowRight } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React, { useState } from 'react';
+import CalendarDays from './calendardays';
+import './calendar.css';
+
+export function Calendar() {
+  const weekdays = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+  const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
```

Comment:
extremely minor optimization would be to move these constants outside of your functional component.

I believe on each render, these values will be reinitialized. It'll probably never matter, but if you can simplify, lets simplify